### ISSUE TITLE
ENH: implement handling of class_weights param

### ISSUE THREAD

**Issue by adriangb** [adriangb/scikeras#52]

**Comment by adriangb** [comment]
It looks like for single-output models, this should be straightforwardish. The main complicating factor is that we're going to have to track this through our encode/decode strategy.

For multi-output models, this is a lot more complicated and may require a custom loss function [link](https://datascience.stackexchange.com/questions/41698/how-to-apply-class-weight-to-a-multi-output-model). I asked in TF (https://github.com/keras-team/keras/issues/4735#issuecomment-696363490) if there is an easier solution for multi-output models, but I get the feeling that there is not.

**Comment by adriangb** [comment]
I'm thinking we'll probably want to do this using https://scikit-learn.org/stable/modules/generated/sklearn.utils.class_weight.compute_sample_weight.html instead of relying on Keras. We'll just apply class_weights first then sample_weights.
